nixos/modules/services/web-servers/phpfpm/default.nix

```diff
@@ -199,21 +198,55 @@ in {
         type = types.attrsOf (types.submodule poolOpts);
         default = {};
         example = literalExample ''
-         {
-           mypool = {
-             user = "php";
-             group = "php";
-             phpPackage = pkgs.php;
-             settings = '''
-               "pm" = "dynamic";
-               "pm.max_children" = 75;
-               "pm.start_servers" = 10;
-               "pm.min_spare_servers" = 5;
-               "pm.max_spare_servers" = 20;
-               "pm.max_requests" = 500;
-             ''';
-           }
-         }'';
+          {
+            web1 = {
+              user = "nginx";
+              group = "nginx";
+              phpPackage = pkgs.php72;
+              settings = {
+                "listen.owner" = "nginx";
+                "listen.group" = "nginx";
+                "pm" = "dynamic";
+                "pm.max_children" = 75;
+                "pm.start_servers" = 10;
+                "pm.min_spare_servers" = 5;
+                "pm.max_spare_servers" = 20;
+                "pm.max_requests" = 500;
+              };
+            }
+
+            web2 = {
+              user = "wwwrun";
+              group = "wwwrun";
+              phpPackage = pkgs.php73;
+              settings = {
+                "listen.owner" = "wwwrun";
+                "listen.group" = "wwwrun";
+                "pm" = "dynamic";
+                "pm.max_children" = 75;
+                "pm.start_servers" = 10;
+                "pm.min_spare_servers" = 5;
+                "pm.max_spare_servers" = 20;
+                "pm.max_requests" = 500;
+              };
+            }
+
+            web3 = {
+              user = "php";
+              group = "php";
+              settings = {
+                "listen" = "9000";
+                "listen.allowed_clients" = "127.0.0.1, 129.0.0.223, 129.0.0.48";
+                "pm" = "dynamic";
+                "pm.max_children" = 75;
+                "pm.start_servers" = 10;
+                "pm.min_spare_servers" = 5;
+                "pm.max_spare_servers" = 20;
+                "pm.max_requests" = 500;
+              };
+            };
+          }
+        '';
```

Comment:
@grahamc Please let me know if these examples sufficiently clarify the `user` and `group` properties for you.

As mentioned in https://github.com/NixOS/nixpkgs/pull/65706#issuecomment-549077995 specifying a `user` and `group` was always required, and the only difference now is that the user gets a build time failure, as opposed to runtime.

Reply:
yeah, an example with `settings.listen.owner`/`settings.listen.group` = `nginx`, but `user`/`group`being different (at least `user`, maybe `user`=`php1`, `group`=`nginx`) seems to make sense.